util/fibers/detail/fiber_interface.cc

```diff
@@ -290,6 +291,20 @@ ctx::fiber_context FiberInterface::SwitchTo() {
   });
 }
 
+void FiberInterface::PrintStackTrace() {
+  entry_ = std::move(entry_).resume_with([name = name_](ctx::fiber_context&& c) {
+    std::string stacktrace = GetStacktrace(false);
```

Comment:
I'd default to true actually, as I imagine that in most cases that we print stack traces we won't care too much about performance

Reply:
It's reeeeeally slow though

Reply:
abseil is faster, so enabled everywhere.